tests/tests_idempotancy.yml

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure that the role is idempotent
+  hosts: all
+  vars:
+    mssql_accept_microsoft_odbc_driver_17_for_sql_server_eula: true
+    mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
+    mssql_accept_microsoft_sql_server_2019_standard_eula: true
+  tasks:
+    - name: Ensure that the role runs on a fresh host with required parameters
+      include_role:
+        name: linux-system-roles.mssql
+      vars:
+        mssql_password: "p@55w0rD"
+        mssql_edition: Evaluation
```

Comment:
You should have some task to verify settings - better yet, a file like `tests/tasks/verify_settings.yml` that you can reuse like this:
```yaml
- name: verify settings
  include_tasks: tasks/verify_settings.yml
  vars:
    __verify_mssql_password: "p@55w0rD"
    __verify_mssql_edition: Evaluation
```

Reply:
I'll try to implement this now

Reply:
Done